packages/yarnpkg-shell/tests/shell.test.ts

```diff
@@ -657,89 +674,89 @@ describe(`Shell`, () => {
 
     describe(`Using variables`, () => {
       it(`should support using environment variables`, async () => {
-        await expect(bufferResult(
+        await expectResult(bufferResult(
           `echo $FOOBAR`,
           [],
           {env: {FOOBAR: `hello world`}},
-        )).resolves.toMatchObject({
+        ), {
           stdout: `hello world\n`,
         });
       });
 
       it(`should support default arguments via \${ARG:-...}`, async () => {
-        await expect(bufferResult(
+        await expectResult(bufferResult(
           `echo "\${DOESNT_EXIST:-hello world}"`,
-        )).resolves.toMatchObject({
+        ), {
           stdout: `hello world\n`,
         });
       });
 
       it(`should support default arguments via \${N:-...}`, async () => {
-        await expect(bufferResult(
+        await expectResult(bufferResult(
           `echo "\${1:-hello world}"`,
           [],
-        )).resolves.toMatchObject({
+        ), {
           stdout: `hello world\n`,
         });
       });
 
       it(`should support empty default arguments`, async () => {
-        await expect(bufferResult(
+        await expectResult(bufferResult(
           `echo "foo\${DOESNT_EXIST:-}bar"`,
-        )).resolves.toMatchObject({
+        ), {
           stdout: `foobar\n`,
         });
       });
 
       it(`should support alternative arguments via \${ARG:+...}`, async () => {
-        await expect(bufferResult(
+        await expectResult(bufferResult(
           `echo "\${FOOBAR:+hello world}"`,
           [],
           {env: {FOOBAR: `goodbye world`}},
-        )).resolves.toMatchObject({
+        ), {
           stdout: `hello world\n`,
         });
 
-        await expect(bufferResult(
+        await expectResult(bufferResult(
           `echo "\${FOOBAR:+hello world}"`,
-        )).resolves.toMatchObject({
+        ), {
           stdout: ``,
```

Comment:
nit: is this `\n`? (I'm not very sure 🙈)

```diff
- stdout: ``,
+ stdout: `\n`,
```

Reply:
I modified this test in another PR: https://github.com/yarnpkg/berry/pull/4959/commits/ca63766bdef6774ee70975b544dfe37121c2167e

make the https://github.com/yarnpkg/berry/pull/4959 test pass🥺